[2023-06-14T04:44:54.167+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: endToend_user_processing.is_api_available manual__2023-06-14T04:44:52.140702+00:00 [queued]>
[2023-06-14T04:44:54.174+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: endToend_user_processing.is_api_available manual__2023-06-14T04:44:52.140702+00:00 [queued]>
[2023-06-14T04:44:54.175+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2023-06-14T04:44:54.177+0000] {taskinstance.py:1363} INFO - Starting attempt 1 of 1
[2023-06-14T04:44:54.178+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2023-06-14T04:44:54.186+0000] {taskinstance.py:1383} INFO - Executing <Task(HttpSensor): is_api_available> on 2023-06-14 04:44:52.140702+00:00
[2023-06-14T04:44:54.192+0000] {standard_task_runner.py:55} INFO - Started process 741 to run task
[2023-06-14T04:44:54.196+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'endToend_user_processing', 'is_api_available', 'manual__2023-06-14T04:44:52.140702+00:00', '--job-id', '22', '--raw', '--subdir', 'DAGS_FOLDER/user_processing.py', '--cfg-path', '/tmp/tmputu5a25h']
[2023-06-14T04:44:54.199+0000] {standard_task_runner.py:83} INFO - Job 22: Subtask is_api_available
[2023-06-14T04:44:54.257+0000] {task_command.py:376} INFO - Running <TaskInstance: endToend_user_processing.is_api_available manual__2023-06-14T04:44:52.140702+00:00 [running]> on host eccc8785320d
[2023-06-14T04:44:54.310+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=endToend_user_processing
AIRFLOW_CTX_TASK_ID=is_api_available
AIRFLOW_CTX_EXECUTION_DATE=2023-06-14T04:44:52.140702+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-06-14T04:44:52.140702+00:00
[2023-06-14T04:44:54.311+0000] {http.py:120} INFO - Poking: api/
[2023-06-14T04:44:54.321+0000] {base.py:71} INFO - Using connection ID 'user_api' for task execution.
[2023-06-14T04:44:54.327+0000] {http.py:148} INFO - Sending 'GET' to url: https://randomuser.me/api/
[2023-06-14T04:44:54.834+0000] {base.py:213} INFO - Success criteria met. Exiting.
[2023-06-14T04:44:54.847+0000] {taskinstance.py:1406} INFO - Marking task as SUCCESS. dag_id=endToend_user_processing, task_id=is_api_available, execution_date=20230614T044452, start_date=20230614T044454, end_date=20230614T044454
[2023-06-14T04:44:54.902+0000] {local_task_job.py:164} INFO - Task exited with return code 0
[2023-06-14T04:44:54.933+0000] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
